frontend/ui/screensaver.lua

```diff
@@ -264,6 +264,8 @@ function Screensaver:show(event, fallback_message)
         return
     end
 
+    -- These historical defaults are image-oriented.
+    -- That'll bite us in the ass later on, both for "disable" mode and show_message...
```

Comment:
(My english sucks, but is "bite us _in_ the ass" correct ? I don't see how that's physically possible, but well, I haven't seen everything :)

Reply:
It's grammatically weird, but the idiom is indeed frozen that way ^^.

(I'm going to refactor ScreenSaver's show in two to try to make the whole thing less kludgey).